lib/PuppeteerSharp.Tests/PublishSingleFileTests/PublishSingleFileTests.cs

```diff
@@ -0,0 +1,96 @@
+using System.Diagnostics;
+using System.IO;
+using System.Threading.Tasks;
+using PuppeteerSharp.Tests.Attributes;
+using Xunit;
+
+namespace PuppeteerSharp.Tests.SingleFileDeployment
+{
+    public class PublishSingleFileTests
+    {
+        [SkipNonWindowsFact]
+        public void ShouldWork()
+        {
+            var tempPath = Path.GetTempPath();
+            var actualFilePath = Path.Combine(tempPath, $"google.jpg");
+            var actualWindowsBinary = DotnetPublishSingleFile("PuppeteerSharp.Tests.SingleFileDeployment");
+
+            DeleteIfExists(actualFilePath);
+
+            using var process = new Process
+            {
+                StartInfo = new ProcessStartInfo
+                {
+                    FileName = actualWindowsBinary,
+                    RedirectStandardOutput = true,
+                    RedirectStandardError = true,
+                    UseShellExecute = false,
+                    CreateNoWindow = true,
+                    WorkingDirectory = tempPath
+                }
+            };
+
+            process.Start();
+            var outputResult = GetStreamOutput(process.StandardOutput);
+            var errorResult = GetStreamOutput(process.StandardError);
+            var isExited = process.WaitForExit(80000);
+
+            if (!isExited)
+            {
+                process.Kill();
+            }
+
+            Assert.True(isExited);
+
+            Assert.True(File.Exists(actualFilePath), $"StdOut: {outputResult}\nStdErr: {errorResult}\n");
+        }
+
+        private static string GetStreamOutput(StreamReader stream)
+        {
+            var outputReadTask = Task.Run(() => stream.ReadToEnd());
+
+            return outputReadTask.Result;
+        }
+
+        private static string DotnetPublishSingleFile(string projectName)
+        {
+            var absolutePath = Path.GetFullPath("../../../../" + projectName);
```

Comment:
Let's use `Path.Join` here.

Reply:
There is no Path.Join in net48. Used Path.Combine.